Dockerfile

```diff
@@ -312,13 +312,14 @@ FROM ubuntu:16.04
 MAINTAINER Seth Fowler <sfowler@barefootnetworks.com>
 ARG DEBIAN_FRONTEND=noninteractive
 ARG MAKEFLAGS=-j2
-ENV CCACHE_RUNTIME_DEPS libmemcached-dev
-ENV SCAPY_VXLAN_RUNTIME_DEPS python-minimal
-ENV PTF_RUNTIME_DEPS libpcap-dev python-minimal tcpdump
-ENV NNPY_RUNTIME_DEPS python-minimal
-ENV THRIFT_RUNTIME_DEPS libssl1.0.0 python-minimal
-ENV GRPC_RUNTIME_DEPS python-minimal python-setuptools
-ENV SYSREPO_RUNTIME_DEPS libpcre3 libavl1 libev4 libprotobuf-c1
+# Runtime dependencies
+ARG CCACHE_RUNTIME_DEPS="libmemcached-dev"
+ARG SCAPY_VXLAN_RUNTIME_DEPS="python-minimal"
+ARG PTF_RUNTIME_DEPS="libpcap-dev python-minimal tcpdump"
+ARG NNPY_RUNTIME_DEPS="python-minimal"
+ARG THRIFT_RUNTIME_DEPS="libssl1.0.0 python-minimal"
+ARG GRPC_RUNTIME_DEPS="python-minimal python-setuptools"
+ARG SYSREPO_RUNTIME_DEPS="libpcre3 libavl1 libev4 libprotobuf-c1"
 RUN apt-get update && \
```

Comment:
not a strong preference but we could also just set these in the `RUN` command to avoid defining args:
```bash
RUN CCACHE_RUNTIME_DEPS = ... && \
        ...
        apt-get update && \
        ...
```

Reply:
Addressed in cd5eb4d.